app/src/sharedTest/java/org/oppia/android/app/testing/TickIconTest.kt

```diff
@@ -0,0 +1,274 @@
+import org.oppia.android.domain.question.QuestionCountPerTrainingSession
+import org.oppia.android.domain.question.QuestionTrainingSeed
+import org.oppia.android.domain.topic.FRACTIONS_SKILL_ID_0
+import org.oppia.android.domain.topic.PrimeTopicAssetsControllerModule
+import org.oppia.android.testing.CoroutineExecutorService
+import org.oppia.android.testing.TestAccessibilityModule
+import org.oppia.android.testing.TestCoroutineDispatchers
+import org.oppia.android.testing.TestDispatcherModule
+import org.oppia.android.testing.TestLogReportingModule
+import org.oppia.android.testing.profile.ProfileTestHelper
+import org.oppia.android.util.caching.testing.CachingTestModule
+import org.oppia.android.util.gcsresource.GcsResourceModule
+import org.oppia.android.util.logging.LoggerModule
+import org.oppia.android.util.logging.firebase.FirebaseLogUploaderModule
+import org.oppia.android.util.parser.GlideImageLoaderModule
+import org.oppia.android.util.parser.HtmlParserEntityTypeModule
+import org.oppia.android.util.parser.ImageParsingModule
+import org.oppia.android.util.threading.BackgroundDispatcher
+import org.robolectric.annotation.Config
+import org.robolectric.annotation.LooperMode
+import javax.inject.Inject
+import javax.inject.Singleton
+
+private val SKILL_ID_LIST = listOf(FRACTIONS_SKILL_ID_0)
+
+/** Tests for [QuestionPlayerActivity]. */
+@RunWith(AndroidJUnit4::class)
+@LooperMode(LooperMode.Mode.PAUSED)
+@Config(application = TickIconTest.TestApplication::class, qualifiers = "port-xxhdpi")
+class TickIconTest {
+  // TODO(#503): add tests for QuestionPlayerActivity (use StateFragmentTest for a reference).
+  // TODO(#1273): add tests for Hints and Solution in Question Player.
+
+  @Inject
+  lateinit var testCoroutineDispatchers: TestCoroutineDispatchers
+
+  @Inject
+  lateinit var profileTestHelper: ProfileTestHelper
+
+  @Inject
+  lateinit var context: Context
+
+  @Inject
+  @field:BackgroundDispatcher
+  lateinit var backgroundCoroutineDispatcher: CoroutineDispatcher
+
+  @Before
+  fun setUp() {
+    setUpTestApplicationComponent()
+    testCoroutineDispatchers.registerIdlingResource()
+    profileTestHelper.initializeProfiles()
+    FirebaseApp.initializeApp(context)
+
+    // Initialize Glide such that all of its executors use the same shared dispatcher pool as the
+    // rest of Oppia so that thread execution can be synchronized via Oppia's test coroutine
+    // dispatchers.
+    val executorService = MockGlideExecutor.newTestExecutor(
+      CoroutineExecutorService(backgroundCoroutineDispatcher)
+    )
+    Glide.init(
+      context,
+      GlideBuilder().setDiskCacheExecutor(executorService)
+        .setAnimationExecutor(executorService)
+        .setSourceExecutor(executorService)
+    )
+  }
+
+  @After
+  fun tearDown() {
+    testCoroutineDispatchers.unregisterIdlingResource()
+  }
+
+  @Test
+  fun testChooseCorrectAnswer_isTickVisibleCompletely() {
+    launchForSkillList(SKILL_ID_LIST).use {
```

Comment:
@BenHenning @MohamedMedhat1998 
 Current behaviour - 
The test is passing on Robolectric as well as espresso with new changes done in the PR.
The test is failing on Espresso but passing unexpectedly on Robolectric without new changes.

Expected behaviour - 
The test should pass on Robolectric as well as espresso with new changes done in the PR.
The test should fail on Espresso and Robolectric without new changes.

Reply:
On Espresso, is this actually leading to a sufficiently long answer that cuts off the tick?

It may also depend on display density. What device/emulator image are you using to run the test? Robolectric is being configured here for port-xxhdpi which is quite large--it's possible that the answer just isn't long enough to cause a cut-off on such a large display density.

Reply:
@BenHenning Yes it is leading to a sufficiently long answer that cuts off the tick, that's why tests are failing on espresso when I tested them in a new branch and it is passing on espresso when I tested it in the current branch. 
But the test is passing on robolectric in both the branches, which should not happen.

Reply:
>  The test is failing on Espresso but passing unexpectedly on Robolectric without new changes.

Perhaps try using specific screen sizes as qualifiers like those we have in `PlayerSplitScreenTesting`